apps/api/src/purchase-orders/purchase-orders.service.ts

```diff
@@ -0,0 +1,17 @@
+import {Injectable} from '@nestjs/common';
+import {PurchaseOrders} from "@prisma/client";
+import {PrismaService} from "../prisma.service";
+
+@Injectable()
+export class PurchaseOrdersService {
+  constructor(private prisma: PrismaService) {
+  }
+
+  async findAll(): Promise<PurchaseOrders[]> {
```

Comment:
It seems like this is an implementation with a default sort order.  Couldn't you just use the `findAllSorted` and pass the desired sorting configuration to that?

Reply:
Yeah, that is actually one thing I was thinking about afterwords. A better approach would be to just have one api call with a default sortedBy option. If the option is provided then we sort by that, otherwise we sort by delivery date.